apps/pipeline/lib/pipeline/writer/table_writer/statement/insert.ex

```diff
@@ -24,11 +19,11 @@ defmodule Forklift.Messages.Statement do
       |> Enum.map(&to_row_string/1)
       |> Enum.join(",")
 
-    ~s|insert into "#{schema.system_name}" (#{columns_fragment}) values #{data_fragment}|
+    {:ok, ~s|insert into "#{config.table}" (#{columns_fragment}) values #{data_fragment}|}
```

Comment:
is there a reason `Create.compose/2` is returning the create statement as a string but `Insert.compose/2` is returning an :ok tuple?

Reply:
Yeah this is an artifact of how Forklift was before. I plan to fix these, but in another PR.